shared/src/main/scala/pl/metastack/metatime/Pattern.scala

```diff
@@ -1,6 +1,6 @@
 package pl.metastack.metatime
 
-case class Pattern(parts: Seq[DateTime => String] = Nil) {
+case class Pattern(parts: Seq[Component => String] = Nil) {
```

Comment:
Actually if I change it back to DateTime then I cant invoke format function for Time instance for following testcase:
    `val time = Time(11, 1, 0, 0)`
    `assert(time.format(Pattern.DefaultTime) == "11:01 AM")`